PerformanceTests/JetStream2/JetStreamDriver.js

```diff
@@ -139,7 +139,7 @@ function geomean(values) {
 }
 
 function toScore(timeValue) {
-    return 5000 / timeValue;
+    return 5000 / Math.max(timeValue, 1);
```

Comment:
Can we make this .5 so there's a difference between 0 and 1?

Reply:
Thanks Keith! This was discussed offline (Mark's comment from last night).

Reply:
I think sticking with 1 makes sense due to `Date.now()`'s resolution being low.  There's not much value in going with a lower number.